api/proto/teleport/legacy/types/types.proto

```diff
@@ -7373,6 +7373,11 @@ message PluginOktaSyncSettings {
   // synchronized users. This is allows for a more advanced RBAC setup where not all
   // Okta-originated users are allowed request all Okta-originated resources.
   bool disable_assign_default_roles = 13;
+
+  // TimeBetweenImportsSeconds controls the time between Okta syncs. I.e. importing Okta users, apps and
+  // groups to teleport. This doesn't affect how quickly Teleport changes are propagated to Okta if
+  // bidirectional sync is enabled. The default value is 1800 (30 minutes).
+  int64 time_between_imports_seconds = 14;
```

Comment:
If you wonder why this isn't Duration type, the short answer is I had problems with JSON encoding. Details in [Slack](https://gravitational.slack.com/archives/C0DF0TPMY/p1758734309832139).

Reply:
WDYT about sync_interval for a name?